### PR TITLE
[ESI-10454] Handling nullable attributes in update and create queries

### PR DESCRIPTION
**Pull request by Zimnx**

### CODE REVIEW

File: db/sql/sql_test.go [hunk 1]
```diff
@@ -173,6 +177,58 @@ var _ = Describe("Sql", func() {
...
+					err := tx.Update(resource)
+					Expect(err).ToNot(BeNil())
+				})
+
+				It("Setting null to nullable field", func() {
+					resource := extractSingleResource()
+					delete(resource.Data(), "description")
+
+					err := tx.Update(resource)
+					Expect(err).To(BeNil())
```
Comment (p-kozlowski) [inline db/sql/sql_test.go]: no error, but let's check if the resource was updated as expected

File: db/sql/sql_test.go [hunk 2]
```diff
@@ -173,6 +177,58 @@ var _ = Describe("Sql", func() {
...
+					Expect(err).ToNot(BeNil())
+				})
+
+				It("Setting null to nullable field", func() {
+					resource := extractSingleResource()
+					resource.Data()["id"] = "id_1"
+					delete(resource.Data(), "description")
+
+					err := tx.Create(resource)
+					Expect(err).To(BeNil())
```
Comment (p-kozlowski) [inline db/sql/sql_test.go]: as above

**Comment by nati** [comment]
LGTM waiting CI

**Comment by Zimnx** [comment]
@nati CI already passed.

**Comment by nati** [comment]
we need running CI again after update..

**Pull request merged**